api.md

```diff
@@ -1685,53 +1686,41 @@ GROUP BY <time_bucket( <const_value>, <partition_col_of_hypertable> ),
 |---|---|---|
 |`timescaledb.continuous`|||
 |**Description**|**Type**|**Default**|
-|If timescaledb.continuous is not specified, then this is a regular PostgresSQL view. | `BOOLEAN` ||
+|If timescaledb.continuous is not specified, then this is a regular PostgresSQL materialized view. | `BOOLEAN` ||
 
 #### Optional `WITH` clause options [](continuous_aggregate-create_view-with-optional)
 
 |**Name**|||
 |---|---|---|
-|`timescaledb.refresh_lag`|||
-|**Description**|**Type**|**Default**|
-| Refresh lag controls the amount by which the materialization will lag behind the current time. The continuous aggregate view lags behind by `bucket_width` + `refresh_lag` value. `refresh_lag` can be set to positive and negative values. | Same datatype as the `bucket_width` argument from the `time_bucket` expression.| The default value is twice the bucket width (as specified by the `time_bucket` expression).|
-|   |   |   |
-|**Name**|||
-|`timescaledb.refresh_interval`|||
-|**Description**|**Type**|**Default**|
-| Refresh interval controls how often the background materializer is run. Note that if `refresh_lag` is set to `-<bucket_width>`, the continuous aggregate will run whenever new data is received, regardless of what the `refresh_interval` value is. | `INTERVAL`|By default, this is set to twice the bucket width (if the datatype of the bucket_width argument from the `time_bucket` expression is an `INTERVAL`), otherwise it is set to 12 hours.|
-|   |   |   |
-|**Name**|||
 |`timescaledb.materialized_only`|||
 |**Description**|**Type**|**Default**|
 | Return only materialized data when querying the continuous aggregate view. | `BOOLEAN` | false |
```

Comment:
Should this section have any pointer to [real-time aggregates][real-time-aggregates] ?  I don't think the reader just looking up this API spec will understand this term.  If so, where should the pointer go?

Reply:
Added a pointer to the section on real-time aggregates.